### ISSUE TITLE
Add default text to first line of Data Elements

### ISSUE THREAD

**Issue by AngelaFaulding** [NHSDigital/DataDictionaryPublication#381]
All Data Elements in the NHS Data Model and Dictionary start with:
DATA ELEMENT NAME is the same as attribute ATTRBUTE NAME.

James can set this up to be added automatically in Mauro.
This will combine the Data Element name with the Data Element linked attribute name.
This wouldn't appear immediately, but after the linked attribute has been added in the "NHS Data Dictionary - Element" profile.

**Comment by AngelaFaulding** [comment]
15/2/22 - discussed with James yesterday. Having extra text is fine. The generated text will display in the Mauro preview and DITA output but not in Mauro.

**Comment by AngelaFaulding** [comment]
16/05/22 - work completed by James. Ticket moved.

**Comment by AngelaFaulding** [comment]
Discussion with James 8/8/22.
This is working but will cause work if extra text is required, e.g. ABBREVIATED MENTAL TEST SCORE  is the  PERSON SCORE  where the ASSESSMENT TOOL  is 'Abbreviated Mental Test Score'.
It would help if we could add text to the standard text.

**Comment by AngelaFaulding** [comment]
22/8/22 - Further to comment 8/8/22.
We have agreed that there will be a tick box to add the standard text.
If text is required below the standard text, add the text and it will be shown as a new paragraph.
If not ticked, we will add the standard text and extra text on that line followed by a new paragraph if required.
New new paragrah

**Comment by jamesrwelch** [comment]
The changes outlined in the last comment are still pending, and once they've been made, data will need to be re-ingested into the Oxford test instance.

**Comment by AngelaFaulding** [comment]
James thinks this is now working.

**Comment by AngelaFaulding** [comment]
@jamesrwelch - I am looking at Data Elements. Will this be included in the next release as it doesn't appear to be working, even though it is in the "Ready for Testing" column.
I tested a Data Element with incorrect text and the Orchestrator only displays the text I added.

![image](https://github.com/NHSDigital/DataDictionaryPublication/assets/62995711/20a2d79f-6706-4e6c-b158-3fd0d669b896)

**Comment by jamesrwelch** [comment]
I expect it's correct in the DITA generation but not in the orchestrator.  I'll take a look

**Comment by AngelaFaulding** [comment]
James thinks this can be done.

**Comment by AngelaFaulding** [comment]
16/07/2024 - James thinks it defaults to is the same as ATTRIBUTE if there is no other text. James will check.

**Comment by AngelaFaulding** [comment]
23/07/2024 - James will check this but hasn't had time this week.

**Comment by AngelaFaulding** [comment]
29/07/24 - James is still looking into this and should be able to do this for migration.

**Comment by AngelaFaulding** [comment]
James will investigate to see if it can be resolved for migration.

**Comment by AngelaFaulding** [comment]
Agreed with James that the text will default to: DATA ELEMENT NAME is the same as attribute ATTRBUTE NAME

If this is not required, James will add a tick box that will remove this so other text can be put in it's place.

The tick box will be "Suppress default first sentence".

**Comment by jamesrwelch** [comment]
This is now working.  But due to an error in the ingest, will need careful testing in UAT - should work correctly in DEV

**Comment by AngelaFaulding** [comment]
This will not work until #673 is working
